nvm.sh

```diff
@@ -2428,6 +2603,18 @@ nvm() {
         nvm_echo '  nvm which [--silent] [current | <version>]  Display path to installed node version. Uses .nvmrc if available'
         nvm_echo '  nvm cache dir                               Display path to the cache directory for nvm'
         nvm_echo '  nvm cache clear                             Empty cache directory for nvm'
+        nvm_echo '  nvm set-colors [<color codes>]              Set five text colors using format "yMeBg". Available when supported.'
```

Comment:
is it "nvm set-colors" or "nvm --set-colors"? below implies it's with the dash (probably without the dash is better)

Reply:
Oh, good catch, thanks! Change made.